src/Microsoft.Win32.Primitives/src/System/ComponentModel/Win32Exception.cs

```diff
@@ -58,29 +58,15 @@ public Win32Exception(string message) : this(Marshal.GetLastWin32Error(), messag
         /// </devdoc>
         public Win32Exception(string message, Exception innerException) : base(message, innerException)
         {
-            nativeErrorCode = Marshal.GetLastWin32Error();
+            _nativeErrorCode = Marshal.GetLastWin32Error();
         }
 
-        protected Win32Exception(SerializationInfo info, StreamingContext context) : base(info, context)
-        {
-            throw new PlatformNotSupportedException();
-        }
-
-        public override void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
@stephentoub had pointed out that many exceptions don't override GetObjectData and thus don't throw PNSE. We didn't want them to be different.